e2e/cypress/integration/sites-and-applications.spec.js

```diff
@@ -2,9 +2,11 @@ describe('Sites and Applications', () => {
   before(() => {
     // Reset the database
     cy.task('db:seed')
+    cy.loginTestIDP()
   })
+
   beforeEach(() => {
-    // Make sure the beta cookie is set
+    cy.preserveLoginCookies()
     cy.visit('/joinbeta')
```

Comment:
Is this where we'd use the `cy.joinBeta()` helper you mentioned in the PR notes, or is that for something else?